src/Features/Core/Portable/LanguageServices/SymbolDisplayService/AbstractSymbolDisplayService.AbstractSymbolDescriptionBuilder.cs

```diff
@@ -243,20 +243,58 @@ void AddValueDocumentationParts(ISymbol symbol, IDocumentationCommentFormattingS
 
             private void AddExceptions(ISymbol symbol)
             {
-                var exceptionTypes = symbol.GetDocumentationComment(GetCompilation(), expandIncludes: true, expandInheritdoc: true).ExceptionTypes;
+                const int maxTextSize = 100;
+                var documentationComment = symbol.GetDocumentationComment(GetCompilation(), expandIncludes: true, expandInheritdoc: true);
+                var exceptionTypes = documentationComment.ExceptionTypes;
+                var formatter = Services.GetLanguageServices(_semanticModel.Language).GetRequiredService<IDocumentationCommentFormattingService>();
                 if (exceptionTypes.Any())
                 {
-                    var parts = new List<SymbolDisplayPart>();
+                    var parts = new List<TaggedText>();
                     parts.AddLineBreak();
                     parts.AddText(WorkspacesResources.Exceptions_colon);
+                    var textSize = 0;
                     foreach (var exceptionString in exceptionTypes)
                     {
-                        parts.AddRange(LineBreak());
-                        parts.AddRange(Space(count: 2));
-                        parts.AddRange(AbstractDocumentationCommentFormattingService.CrefToSymbolDisplayParts(exceptionString, _position, _semanticModel));
+                        var checkedParts = new List<TaggedText>();
+                        checkedParts.AddRange(LineBreak().ToTaggedText());
+                        checkedParts.AddRange(Space(count: 2).ToTaggedText());
+                        checkedParts.AddRange(AbstractDocumentationCommentFormattingService.CrefToSymbolDisplayParts(exceptionString, _position, _semanticModel).ToTaggedText());
+                        var exceptionTexts = documentationComment.GetExceptionTexts(exceptionString);
+                        if (exceptionTexts.Length > 0)
+                        {
+                            var exceptionStr = $": {exceptionTexts[0]}";
+                            checkedParts.AddRange(formatter.Format(exceptionStr, symbol, _semanticModel, _position, ISymbolExtensions2.CrefFormat, CancellationToken));
+                        }
+
+                        for (var itemIndex = 0; itemIndex < checkedParts.Count; itemIndex++)
```

Comment:
This loop seems like overkill for what is needed. The chances of the exception type being longer than 100 chars is low, and if by some chance it is, it doesn't really make sense to not show it anyway, as that would be removing all useful info. Seems to me it would be much easier to just ensure `exceptionStr` above is truncated at some length if its too long, and leave it at that. Maybe 70 chars?

Reply:
I placed 100 originally as a sane default to try to cover all the possible exception texts that might be in the default framework docs in dotnet/runtime, dotnet/wpf, dotnet/winforms, dotnet/aspnetcore in case. I can reduce it to 70 though.

Reply:
You was right as well, the loop is not needed. Removing it for another option that does not over truncate.

Reply:
There is of course the other question or whether there should be any truncation at all. Do other parts of Quick Info do it?

Reply:
I do not think they do, but then there is also the issue with it not filling the entire text window as well.

Reply:
I think you are right with the whole truncation thing, what could be done instead is for it to count the number of text in each exception documentations and then use that to calculate how many exceptions should be displayed on the IDE side of things.